docs/handbook/common-tasks.md

```diff
@@ -45,6 +45,87 @@ Print versions are also available.
 
 ## Basic system administration
 
+### Basic system information
+
+#### System processes
+
+```markdown
+prstat
+```
+
+This command provides a host of information on all processes running on the system.
+Some of the information provided is as follows:
+
+- percentage of CPU used by each process
+- amount of memory consumed by each process
+- unique id of each process (which can, for example, be used to stop the process)
+
+#### Disk usage
+
+```markdown
+df -h
+```
+
+Provides information on disk size, amount of space used and available free space
+for all attached storage devices. The `-h` option reports this information in
+human readable format.
+
+#### Largest files in a directory
+
+Go to the directory using the `cd` command and issue the following command:
+
+```markdown
+du | sort -n
+```
+
+This will list the size of each file in the current directory and all
+subdirectories, starting with the smallest up to the largest files.
+
+#### Who is logged on to the system
+
+```markdown
+listusers
+```
+
+#### List all software packages installed on the system
+
+```markdown
+pkginfo
```

Comment:
This applies to SVR4 deprecated packaging, we use IPS, thus "pkg list" is better choice.